test/integration/faraday_middleware_test.rb

```diff
@@ -22,6 +22,30 @@ def teardown
       Circuitbox.reset
     end
 
+    def open_circuit
+      volume_threshold = Circuitbox['test'].option_value(:volume_threshold)
+      (volume_threshold + 1).times { @connection.get(@failure_url) }
+    end
+
+    def test_failure_count
```

Comment:
I know why those tests are here but I feel like they assert to low level for an integration test. Could we create one like

```
def test_circuit_does_not_open_for_below_threshhold_failed_requests
  5.times { @connection.get(@failure_url) }
  assert_equal @connection.get(@success_url).status, 200
end